.gitlab/cori.yml

```diff
@@ -27,3 +27,26 @@ cori_pr_regression_test:
     - codecov -t $CODECOV_TOKEN
     - source deactivate
     - conda env remove -p $CI_PROJECT_DIR/.conda -y
+
+perlmutter_regression_test:
+  tags: ["bdtest-perlmutter"]
+  stage: regression
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "external_pull_request_event" && $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == $CI_DEFAULT_BRANCH'
```

Comment:
make sure you add the `changes` like you did in https://github.com/buildtesters/buildtest/pull/1358

Reply:
Done